### PR TITLE
Jenkins integration

### PR DESCRIPTION
**Pull request by PizzaLi**
Fixes  ISSUE#xxx

Changes:

1. move cleanning operation into playbook

### CODE REVIEW

File: build/ci/ansible/roles/kubefate/files/clean.sh [hunk 1]
```diff
@@ -1,18 +1,35 @@
 #/bin/bash
 
-clean_when_timeout()
+clean()
 {
   rm -rf ${BASE_DIR}/*
 
-  # Install Kind
+  # Delete kind
```
Comment (jiahaoc1993) [inline build/ci/ansible/roles/kubefate/files/clean.sh:7]: This is not kind deletion, should replace with remove k8s cluster

File: build/ci/ansible/roles/kubefate/files/clean.sh [hunk 2]
```diff
@@ -1,18 +1,35 @@
...
   fi
 
-  echo "exit because of task timeout"
-  exit 1
+  # delete docker containers
+  docker stop $(docker ps -a -q)
+  docker rm $(docker ps -a -q)
+
+  # delete docker images
+  docker rmi -f $(docker images -q)
```
Comment (jiahaoc1993) [inline build/ci/ansible/roles/kubefate/files/clean.sh:19]: I think is it fine to only remove images with `${DOCKER_REGISTRY}federatedai/` prefix

File: build/ci/ansible/roles/kubefate/files/clean.sh (hunk 1 above)
Comment (PizzaLi) [reply build/ci/ansible/roles/kubefate/files/clean.sh]: done

File: build/ci/ansible/roles/kubefate/files/clean.sh (hunk 2 above)
Comment (PizzaLi) [reply build/ci/ansible/roles/kubefate/files/clean.sh]: done

**Review by jiahaoc1993 — APPROVED** [review]

**Pull request merged**